src/components/files/uploadthing-context.tsx

```
@@ -83,7 +83,7 @@ export default function UploadThingProvider({
             return
         }
 
-        return await startUpload(images.map((image) => image.data.image_data.file_data!))
+        return await startUpload(images.map((image) => image.data.imageData.fileData!))
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)